### PR TITLE
Add qemu-system-aarch64 capabilities

### PR DESCRIPTION
**Pull request by ashwing123**
Included a new default config for aarch64 platforms and obtained appropriate paths. Also using project-machine/qcli instead of raharper/qcli.

### CODE REVIEW

File: go.mod [hunk 1]
```diff
@@ -60,3 +60,5 @@ require (
 	gopkg.in/ini.v1 v1.67.0 // indirect
 	gopkg.in/yaml.v3 v3.0.1 // indirect
 )
+
+replace github.com/raharper/qcli => github.com/project-machine/qcli v0.2.0
```
Comment (raharper) [inline go.mod]: Instead of replace here,  just make the change in the required section.

File: pkg/api/qconfig.go [hunk 2]
```diff
@@ -2,22 +2,28 @@ package api
 
 import (
 	"fmt"
+	"github.com/raharper/qcli"
```
Comment (raharper) [inline pkg/api/qconfig.go]:
```suggestion
	"github.com/project-machine/qcli"
```

**Review by raharper — CHANGES_REQUESTED** [review]
Some minor things to tidy up in comments.

Thanks!  It's really gratifying to see how *little* machine needed to bring up arm64 qemu via the qcli changes.

File: pkg/api/qconfig.go (hunk 2 above)
Comment (ashwing123) [reply pkg/api/qconfig.go]: I think the module name in the go.mod of qcli might need to be changed: https://github.com/project-machine/qcli/blob/main/go.mod will submit a PR

Comment (raharper) [reply pkg/api/qconfig.go]: Indeed!

**Comment by ashwing123** [comment]
Fixed above issues :)

**Review by raharper — CHANGES_REQUESTED** [review]
Can you rebase and squash into single commit please?

**Comment by ashwing123** [comment]
Rebased and squashed

**Review by raharper — APPROVED** [review]

**Pull request merged**